### DISCUSSION TITLE
Remove return type from __init__ snippet

### DISCUSSION THREAD

**Discussion by savannahostrowski** [microsoft/pylance-release#2593] [Enhancement]
If I'm in a typeCheckingMode of "off", I don't want to see a return type for function snippets.

![image](https://user-images.githubusercontent.com/8949415/117730846-e84acf80-b1a1-11eb-9239-e79fe3d5db0e.png)

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Reply by Camiloez** [reply]
Any news about it? I'm having the same problem and after wasting some hours... still can't disable it

**Reply by JacobCallahan** [reply]
Same. This is still a pain point.

**Reply by Banta2000** [reply]
Any update?

**Reply by MIRAMAXED** [reply]
Any update?

**Comment by judej** [comment]
This issue has been added to our backlog

**Reply by luabud** [reply]
Here's the issue to track progress: https://github.com/microsoft/pylance-release/issues/5367
